### PR TITLE
Fix `ibm_is_instance` limitations on supported profiles

### PR DESCRIPTION
**Pull request by luisarojas**
> Address limitations of on https://github.com/IBM-Cloud/infracost/pull/199

# Added

* Support for VSI profiles:

  * `bx3dc-16x80`
  * `bx3dc-24x120`
  * `bx3dc-2x10`
  * `bx3dc-32x160`
  * `bx3dc-48x240`
  * `bx3dc-4x20`
  * `bx3dc-64x320`
  * `bx3dc-8x40`
  * `bx3dc-96x480`
  * `cx3dc-128x320`
  * `cx3dc-16x40`
  * `cx3dc-24x60`
  * `cx3dc-2x5`
  * `cx3dc-32x80`
  * `cx3dc-48x120`
  * `cx3dc-4x10`
  * `cx3dc-64x160`
  * `cx3dc-8x20`
  * `cx3dc-96x240`

Note: No service type was found that contains profile `vx2d-44x616`, so it was removed from unit testing

### CODE REVIEW

File: internal/providers/terraform/ibm/testdata/is_instance_test/is_instance_test.tf [hunk 1]
```diff
@@ -127,155 +127,174 @@ resource "ibm_is_dedicated_host_group" "dedicated_host_group" {
...
-    "cx3d-16x40",
-    "cx3dc-16x40",
-    "bx2-16x64",
-    "bx2a-16x64",
-    "bx2d-16x64",
-    "bx3d-16x80",
-    "gx3-16x80x1l4",
+
+    "bx2-128x512",
+    # "bx2-16x64",
```
Comment (hiltol) [inline internal/providers/terraform/ibm/testdata/is_instance_test/is_instance_test.tf:132]: why are some profiles commented out?

**Review by hiltol — CHANGES_REQUESTED** [review]

File: internal/providers/terraform/ibm/testdata/is_instance_test/is_instance_test.tf (hunk 1 above)
Comment (luisarojas) [reply internal/providers/terraform/ibm/testdata/is_instance_test/is_instance_test.tf:132]: The only way I was able to get the unit tests running was to only to smaller batches at a time, so I'm leaving in one of each type... Are there any others in the list we should uncomment?

**Comment by hiltol** [comment]
I don't see the new profiles added in the test appear in the golden file?

**Comment by luisarojas** [comment]
> I don't see the new profiles added in the test appear in the golden file?

You're right, that was missed — updated

**Review by hiltol — APPROVED** [review]

**Pull request merged**